.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto eol=lf
```

Comment:
This seems to resolve the related issue by keeping all the test files LF on windows vs. the one specific one below

Reply:
Thanks for finding out!